go.mod

```diff
@@ -1,22 +1,24 @@
-module github.com/99designs/keyring
+module github.com/cosmos/keyring
```

Comment:
Thanks for pointing this but for now we don't want to change this to avoid imports conflicts in other repos.

Reply:
We will revert this.  There may have been some difficulty with testing cmd/keyring/main.go without the change.  Can we agree that cmd/keyring/main.go does not need to function properly for merge?

Reply:
Yes, that cmd is not 100% right, you can ignore it